src/models/uniqueId.ts

```diff
@@ -1,5 +1,5 @@
-import * as uuid from "uuid";
+import { randomUUID } from "crypto";
 
 export default function uuidNoDashes(): string {
```

Comment:
I saw a few places where we are still performing manual removal of dashes in the code, we can use this function there